app/src/main/java/sdp/moneyrun/ui/game/GameLobbyActivity.java

```diff
@@ -141,6 +141,19 @@ public void onDataChange(@NonNull DataSnapshot snapshot) {
                         String newPlayersMissing = getString(R.string.lobby_player_missing,game.getMaxPlayerCount() - newPlayers.size());
 
                         playersMissing.setText(newPlayersMissing);
+                        GenericTypeIndicator<Boolean> coinIndicator = new GenericTypeIndicator<Boolean>() {
+                        };
+                        if(!game.getHost().equals(user)) {
+                            boolean started = snapshot.child("started").getValue(coinIndicator);
+                            if (started) {
+                                Intent intent = new Intent(getApplicationContext(), MapActivity.class);
```

Comment:
This is an important feature, and as such, it should be tested.

Reply:
agreed will d the tests for it, I was just preparing the PR

Reply:
done

Reply:
Yes sorry, didn't realize, thanks!